common/src/stack/report-system/command/report/system/tests/test_services.py

```diff
@@ -44,8 +44,9 @@ def test_service_enabled_and_running(host, service):
 def test_tftpd_enabled_and_running(host):
 	xinetd = host.service('xinetd')
 	fbtftpd = host.service('fbtftpd')
-	assert xinetd.is_enabled or fbtftpd.is_enabled, "No tftp server is enabled."
-	assert xinetd.is_running or fbtftpd.is_running, "No tftp server is running."
+	tftpsocket = host.service('tftp.socket')
+	assert tftpsocket.is_enabled or xinetd.is_enabled or fbtftpd.is_enabled, "No tftp server is enabled."
```

Comment:
If we're using `systemd` now, why is the `xinetd` service still checked here?

Reply:
So the test will still theoretically run/work against an older stacki if backported.  Dunno if that's a compelling enough reason.

Reply:
Ah, I missed that this was a report system test.